src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -177,6 +183,12 @@ public TracerSettings(IConfigurationSource source)
         /// <seealso cref="ConfigurationKeys.DisabledIntegrations"/>
         public HashSet<string> DisabledIntegrationNames { get; set; }
 
+        /// <summary>
+        /// Gets or sets the AdoNet types to exclude from automatic instrumentation.
+        /// </summary>
+        /// <seealso cref="ConfigurationKeys.AdoNetExcludeTypes"/>
+        public HashSet<string> AdoNetExcludedTypes { get; set; }
```

Comment:
I'm not sure how you all feel about the TracerSetting reading more like a noun "These are the `ExcludedTypes`" vs. the command line argument reading like a command "We must `EXCLUDE_TYPES`". Do you have any strong feelings about this?

Reply:
I'd go for consistency. We already have `DD_DISABLED_INTEGRATIONS` that maps on `DisabledIntegrationNames`, so `EXCLUDED_TYPES` sounds more consistent.

Reply:
+1. We also use the past participle in `{FEATURE}_ENABLED` settings. Consistency 🎉

Reply:
Yep that's good feedback. Done in 1c79ecf664518805771a7cfb6b6ee627a598f0a8